lib/exercism/team.rb

```diff
@@ -88,6 +88,13 @@ def dismiss(username)
     members.delete(user)
   end
 
+  def dismiss_invitation(username)
+    user = User.find_by_username(username)
```

Comment:
Ah, yepp. I agree. @MohanL that's the last change request, then we're ready to pull this in. Once you've made that change would you please rebase and squash your commits?